src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -13,15 +13,103 @@ import HookStore from '../stores/hookStore';
 import ProjectsStore from '../stores/projectsStore';
```

Comment:
more or less this is just me moving the old page into this component and adding a bit better empty state/error handling + feature switch